pkg/cmd/infra/router/template.go

```diff
@@ -54,6 +55,7 @@ type TemplateRouter struct {
 }
 
 func (o *TemplateRouter) Bind(flag *pflag.FlagSet) {
+	flag.StringVar(&o.RouterName, "name", util.Env("ROUTER_SERVICE_NAME", "public"), "The name the router will identify itself with in the route status")
```

Comment:
this is giving two different defaults to the `ROUTER_SERVICE_NAME` env var (see `Complete()`).  I guess that's not going to cause any harm but it's a little confusing.